Dockerfile

```diff
@@ -1,37 +1,33 @@
-FROM registry.opensource.zalan.do/stups/python:3.5-cd28
+FROM registry.opensource.zalan.do/stups/openjdk:1.8.0-222-22
 MAINTAINER Team Aruha, team-aruha@zalando.de
 
-ENV KAFKA_VERSION="1.1.1" SCALA_VERSION="2.12" JOLOKIA_VERSION="1.3.3"
+ENV KAFKA_VERSION="2.3.0" SCALA_VERSION="2.12" JOLOKIA_VERSION="1.3.3"
```

Comment:
`1.6.2` vs `1.3.3`

Reply:
It's nice that we want to update all the dependencies while merging repo. Nothing unpredictable could happen.